src/StabilityCheck.jl

```diff
@@ -52,20 +52,22 @@ include("annotations.jl")
 # Relies on `is_stable_function`.
 # (*) "all" can mean all or exported; cf. `SearchCfg`'s  `exported_names_only`.
 #
-is_stable_module(mod::Module, scfg :: SearchCfg = default_scfg) :: StCheckResults = begin
-    @debug "is_stable_module: $mod"
+const MODULES_SEEN = Set{Module}()
+is_stable_module_aux(mod::Module, root::Module, scfg :: SearchCfg = default_scfg) :: StCheckResults = begin
+    @debug "is_stable_module($mod)"
+    push!(MODULES_SEEN, mod)
```

Comment:
Presumably, `MODULES_SEEN` should be cleaned up in the top-level wrapper (`is_stable_module`)?